tools/fluent/gecko_builtins.md

```diff
@@ -0,0 +1,48 @@
+# Gecko BUILT-INs
+
+Functions can be used in Fluent to format data according to the current language’s rules, or can provide additional data to fine tune the translation.
+
+Fluent includes some [generic built-in functions](http://projectfluent.org/fluent/guide/functions.html#built-in-functions), like `NUMBER` and `DATETIME`. Such built-in functions can be used in placeables and selectors. For example:
```

Comment:
Just quickly say what NUMBER and DATETIME do? And refer to the docs for the detailed options? Maybe just `for number and date formatting, resp.`.

Reply:
Sure